TUTORIAL.md

```diff
@@ -0,0 +1,126 @@
+# Guide to Solving and Reviewing Deli Counter
+
+After running `rspec` you'll see we have seven in total to pass. Let's start with 
+
+### `#line`
+
+```bash
+#line
+    there is nobody in line
+      should say the line is empty (FAILED - 1)
+
+Failures:
+
+  1) Deli Counter #line there is nobody in line should say the line is empty
+     Failure/Error: line(katz_deli)
+     NoMethodError:
+       undefined method `line' for #<RSpec::ExampleGroups::DeliCounter::Line::ThereIsNobodyInLine:0x007fc1f39fdf28>
+```
+To pass this `undefined method 'line'` we have to create a `line` method in our `deli_counter.rb`.
+
+```ruby
+def line
+end
+```
+When we run `rspec --fail-fast`, we'll see the test is still not passing but the error message changed to `wrong number of arguments (1 for 0)`. Our `line` method should accept and `katz_deli` argument.
```

Comment:
Explain this error message again. What do you mean 1 for 0? Also take a look at the specs. How did you know that one input is supposed to be the deli line?